### DISCUSSION TITLE
MikrotikMCP-v0.2.4

### DISCUSSION THREAD

**Discussion by DylanAKing** [DylanAKing/MikrotikMCP#7] [General]
## Changes:
- Added ability to specify ssh port, otherwise it defaults to port 22
- Fixed sample text to re-appear if the user didn't provide any input
- Added semi-colon, blank space, and new line characters to be processed as separators for target input
- Re-organized code
- Minor code slimming in places
- Enabled editing of targets field after loading target from a file

## Known Issues:
- code handling sample text removal and re-entering currently too "fat"; using 2 functions per text box rather than 2 functions for all text boxes

<hr /><em>This discussion was created from the release <a href='https://github.com/DylanAKing/MikrotikMCP/releases/tag/v0.2.4'>MikrotikMCP-v0.2.4</a>.</em>

**Comment by Simonk007** [comment]
Hi Dyklan,
love your work

i tried installing it. Im using python3. i did pip3 install command and all went well.
when i do ./pat/MMCP.py but i get the error below

Traceback (most recent call last):
  File "./MMCP.py", line 8, in <module>
    import tkinter as Tk
ModuleNotFoundError: No module named 'tkinter'

i tried to install tkinter manual but that did not help. 
can you help me out

thx

**Reply by risingmoon** [reply]
what do you get when you go into Python3 shell mode by typing `python3`

```
$ python3
>>> import tkinter
```

Can you also provide OS and Python version.
